src/components/Hero/HomepageHero.tsx

```diff
@@ -66,6 +87,33 @@ export const HomepageHero = () => {
           </m.div>
         </Heading>
       </Container>
+      <div className="su-relative su-z-10 su-h-[100vw] md:su-h-600 xl:su-h-[57vw] su-w-full su-bg-black">
+        <video
```

Comment:
I'll extract this into a component later after finding out more about client's video requirements.